tools/bodyteleop/README.md

```diff
@@ -0,0 +1,49 @@
+# bodyteleop
+
+## Components
+- `web.py` is the tele-operation server that starts automatically when the commabody goes onroad, which can be found at `https://<body-ip>:5000`.
+- `bodyav.py` has all the audio/video webRTC tracks
+- `static/` contains the teleop ui
+- `bodycontrolsd.py` gets all relevant input messages, processes them and sends the final `testJoystick` message (which the body executes).
+
+
+## Running a model remotely
+A very basic requirement to build cool applications on the body is to be able to:
+- Receive image frames from the body to a remote computer
+- Run the model on the frames, stream the outputs to the body via cereal messaging
+- Control the body based on these model outputs
+
+This section walks you through this process, running YOLO as an example.
+
+### 1. Receive images
+```
+# On the remote computer, run 
+./tools/camerastream/compressed_vipc.py --nvidia --cams 2 <body-ip>
+```
+This decodes the `driverEncodeData` packets from the body, and pushes it to a local VisionIPC server.
+
+### 2. Setup cereal bridge
+
+SSH into the body from the remote computer you intent to run models on. Then start cereal bridge using command below. Replace services... with the list of published sockets you intent to share with the body. For the yolo example that would be: bodyReserved1
```

Comment:
these are instructions and snippets to get the laptop -> body bridge working. I've created short scripts which does it for you in similar fashion to compressed_vipc.py (you just supply the body ip), but not sure if it's thats necessary